example/JSON/BSMPT.json

```diff
@@ -0,0 +1,17 @@
+{
+    "FirstLine": "2",
```

Comment:
if we touch this it might be cool to unify the spelling of the options. I would prefere camel case starting with a small letter :P

Reply:
The new argparser is completly case insensitive, so we can do that.

Reply:
The args are changed to `firstLine`etc